frontend/packages/topology/locales/en/topology.json

```
@@ -8,19 +8,19 @@
   "List": "List",
   "Move connector": "Move connector",
   "Delete connector": "Delete connector",
-  "Edit Application grouping": "Edit Application grouping",
+  "Edit application grouping": "Edit application grouping",
   "View all {{size}}": "View all {{size}}",
-  "Select an Application": "Select an Application",
-  "Create Application": "Create Application",
+  "Select an application": "Select an application",
+  "Create application": "Create application",
   "No Application group": "No Application group",
   "Application": "Application",
-  "Warning: the Application grouping already exists.": "Warning: the Application grouping already exists.",
-  "A unique name given to the Application grouping to label your resources.": "A unique name given to the Application grouping to label your resources.",
-  "Select an Application for your grouping or no Application group to not use an Application grouping.": "Select an Application for your grouping or no Application group to not use an Application grouping.",
+  "Warning: the application grouping already exists.": "Warning: the application grouping already exists.",
+  "A unique name given to the application grouping to label your resources.": "A unique name given to the application grouping to label your resources.",
+  "Select an application for your grouping or no application group to not use an application grouping.": "Select an application for your grouping or no application group to not use an Application grouping.",
```

```suggestion
  "Select an application for your grouping or no application group to not use an application grouping.": "Select an application for your grouping or no application group to not use an application grouping.",
```

The key should match the value here. "use an application grouping" instead of "use an Application grouping"

A simple `yarn i18n` should update this correctly, or?

---

updated